libnopegl/src/node_texture.c

```diff
@@ -371,6 +372,43 @@ static int texture_prefetch(struct ngl_node *node)
     ngli_image_init(&s->image, &image_params, &s->texture);
     s->image.rev = s->image_rev++;
 
+    if (s->rtt) {
+        /* Transform the color textures coordinates so it matches how the
+         * graphics context uv coordinate system works regarding render targets */
+        ngli_gpu_ctx_get_rendertarget_uvcoord_matrix(gpu_ctx, s->image.coordinates_matrix);
+
+        int depth_format = NGLI_FORMAT_UNDEFINED;
+        if (s->renderpass_info.features & NGLI_RENDERPASS_FEATURE_STENCIL)
+            depth_format = ngli_gpu_ctx_get_preferred_depth_stencil_format(gpu_ctx);
+        else if (s->renderpass_info.features & NGLI_RENDERPASS_FEATURE_DEPTH)
+            depth_format = ngli_gpu_ctx_get_preferred_depth_format(gpu_ctx);
```

Comment:
I feel like this is the 3rd time I'm seeing this chunk of code, you might want to consider a dedicated function somewhere

Reply:
Will do in the next PR relative to render targets.